File: py/IEC61970/Base/Core/CurveStyle.py
from enum import Enum


class CurveStyle(Enum):
	"""
	Style or shape of curve.
	"""
	# /**
	#  * The Y-axis values are assumed constant until the next curve point and prior to
	#  * the first curve point.
	#  */
	CONSTANT_Y_VALUE = 1
	# /**
	#  * The Y-axis values are assumed to be a straight line between values.  Also known
	#  * as linear interpolation.
	#  */
	STRAIGHT_LINE_Y_VALUES = 2

File: java/IEC61970/Base/Core/CurveStyle.java
package IEC61970.Base.Core;


/**
 * Style or shape of curve.
 * @created 02-Jan-2024 10:39:24 PM
 */
public enum CurveStyle {
	/**
	 * The Y-axis values are assumed constant until the next curve point and prior to
	 * the first curve point.
	 */
	constantYValue,
	/**
	 * The Y-axis values are assumed to be a straight line between values.  Also known
	 * as linear interpolation.
	 */
	straightLineYValues
}